adk-workflows/agents/centos-package-updater/sub_agents/agent.py

```diff
@@ -0,0 +1,143 @@
+from google.adk.agents import Agent
+from typing import Dict, Any
+import os
+
+def get_config() -> Dict[str, Any]:
```

Comment:
I hope we'll be able to do yaml configs in the future so we don't bloat the env variables

but I completely understand this is done for sake of the demo